python/foglamp/services/core/api/scheduler.py

```diff
@@ -533,6 +533,8 @@ def _extract_args(data, curr_value):
         retval, message = await server.Server.scheduler.delete_schedule(uuid.UUID(schedule_id))
 
         return web.json_response({'message': message, 'id': schedule_id})
+    except TypeError:
```

Comment:
@amarendra-dianomic  why its a type error?

Reply:
Because this incident matches closest to the definition of TypeError:
![6 built in exceptions python 2 7 14 documentation](https://user-images.githubusercontent.com/27818158/36775873-c752d03a-1c89-11e8-936a-51e840933350.png)

Reply:
To avoid this misunderstanding, replaced this TypeError with RuntimeWarning. Pl check.